recipes/fftw/all/conandata.yml

```
@@ -1,6 +1,10 @@
 sources:
   "3.3.9":
-    url: "http://www.fftw.org/fftw-3.3.9.tar.gz"
+    url:
+      [
+        "http://www.fftw.org/fftw-3.3.9.tar.gz",
+        "http://fftw.org/fftw-3.3.9.tar.gz",
```

```suggestion
        "https://www.fftw.org/fftw-3.3.9.tar.gz",
        "https://fftw.org/fftw-3.3.9.tar.gz",
```

HTTPS worked for me.

---

Great, let's do that instead.
Btw, how do you do suggest changes? I want to learn how to do that! hahah

---

https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request

At step 6 it lists a +- button. So basically, you click at line that you want to suggest, and click on that button. Also, you can select multiple lines at same, just like I did here.